pandas/core/generic.py

```diff
@@ -9082,6 +9083,9 @@ def _where(
         name="where",
         name_other="mask",
     )
+    @deprecate_nonkeyword_arguments(
+        version="2.0", allowed_args=["self", "cond", "other"]
```

Comment:
Can you put `version=None` please? I know the original PR I opened had this, but then other reviewers preferred to not specify the version number

Reply:
`
    @deprecate_nonkeyword_arguments(
        version=None, allowed_args=["self", "cond", "other"]
    )
`.  Right?  I have updated my pull request for this now

Reply:
How did you reference the code with the line number in by the way? I tried insert quote but am not getting the same view as yours?

Reply:
Looks right, will take a closer look after work!

Like this: https://youtu.be/5IDpsBccqLk